package.json

```diff
@@ -26,15 +26,16 @@
     "@types/mocha": "^9.0.0",
     "@types/node": "^16.11.7",
     "chai": "^4.3.4",
+    "deploy-eip-1820": "github:pooltogether/deploy-eip-1820",
```

Comment:
Use npm package instead of github link https://www.npmjs.com/package/deploy-eip-1820

Reply:
fixed